src/rgw/rgw_admin.cc

```diff
@@ -8355,7 +8355,12 @@ int main(int argc, const char **argv)
 
     formatter->open_object_section("attrs");
     for (iter = other_attrs.begin(); iter != other_attrs.end(); ++iter) {
-      dump_string(iter->first.c_str(), iter->second, formatter.get());
+      bufferlist& bl = iter->second;
+      if (iter->first == RGW_ATTR_OBJ_REPLICATION_TIMESTAMP) {
+        decode_dump<ceph::real_time>("user.rgw.replicated-at", bl, formatter.get());
```

Comment:
this part should be in the for-loop above. this loop over `other_attrs` is for the leftovers that get dumped as strings

Reply:
sorry, i guess the intent is for this to show up under "attrs" with a key like the other xattrs. i think this is fine